editor/src/core/workers/parser-printer/uid-fix.ts

```diff
@@ -71,6 +71,8 @@ const expressionFunctionCallUIDOptic: Optic<JSExpressionFunctionCall, string> =
 const expressionOtherJavaScriptUIDOptic: Optic<JSExpressionOtherJavaScript, string> =
   fromField('uid')
 
+const expressionJSXElementOptic: Optic<JSXElement, string> = fromField('uid')
```

Comment:
I think this is missing a "uid" from the name 
```suggestion
const expressionJSXElementUIDOptic: Optic<JSXElement, string> = fromField('uid')
```

Reply:
I removed it, it was unused.